frontend/system/src/Entities/EntityDetails.tsx

```diff
@@ -81,7 +81,12 @@ export const EntityDetails = () => {
         onNodeToggle={(_, codes: string[]) => setExpanded(codes)}
         sx={{ overflow: 'auto', width: '100%', marginY: '16px' }}
       >
-        <EntityTreeItem showAllCodes={showAllCodes} entity={entity} isRoot />
+        <EntityTreeItem
+          showAllCodes={showAllCodes}
+          entity={entity}
+          isRoot
+          highlightCode={code}
```

Comment:
Feels ok for me to highlight the product, since it is what you selected, especially because it's quite subtle, if we make it more obvious then I agree it's a good idea to skip the product level.